test/coefficients.jl

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2025: Joaquim Garcia, Oscar Dowson and contributors
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+module TestCoefficients
+
+import ModelAnalyzer
+using Test
+using JuMP
+
+function runtests()
+    for name in names(@__MODULE__; all = true)
+        if startswith("$name", "test_")
+            @testset "$(name)" begin
+                getfield(@__MODULE__, name)()
+            end
+        end
+    end
+    return
+end
+
+function test_linear()
+    model = Model()
+    @variable(model, x <= 2e9)
+    @variable(model, y >= 3e-9)
+    @variable(model, z == 4e-9)
+    @constraint(model, x + y <= 4e8)
+    @constraint(model, x + y + 5e7 <= 2)
+    @constraint(model, 7e6 * x + 6e-15 * y + 2e-12 >= 0)
+    @constraint(model, x <= 100)
+    @constraint(model, 0 * x <= 100)
+    @objective(model, Max, 1e8 * x + 8e-11 * y)
+
+    data = ModelAnalyzer.NumericalAnalyzer.analyze(model)
```

Comment:
```suggestion
    data = ModelAnalyzer.analyze(ModelAnalyzer.NumericalAnalyzer(), model)
```
What about this?

Reply:
I saw this was your suggestion. I am not sure yet.

That is why I started the other 2 as well. To check it a commonish api would make sense.

On one hand, there is already an iis api, we have some other stuff to say that may or may not fit on that.

Same for dual feasibility, we already have an api for primal feas check.

I think we need to play a bit with the other 2, to be more sure.

On another front,
If we keep them reasonably separated its better to integrate just one in the future.